<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddComputerDialog</class>
 <widget class="QDialog" name="AddComputerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Add Computer</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>NAME</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_error_add_computer_name">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="input_add_computer_name"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>BUILDING YEAR</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_error_add_computer_building_year">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="input_add_computer_building_year"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>TYPE OF</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_error_add_computer_type_of">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="input_add_computer_type_of"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>BUILT</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="button_radio_add_computer_yes">
       <property name="text">
        <string>yes</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="button_radio_add_computer_no">
       <property name="text">
        <string>no</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_error_add_computer_built">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="button_add_computer_add_computer">
       <property name="text">
        <string>Add computer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_add_computer_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
